docs/usage/shoot_workerless.md

```diff
@@ -1,3 +1,8 @@
+---
+title: Workerless Shoot
+description: `spec.Provider.Workers` being empty defines a Workerless Shoot
```

Comment:
```suggestion
description: What is a Workerless Shoot and how to create one
```
The description should give an idea what is described in the topic instead of summarizing it or giving concrete details. If you feel that my suggestion does not add additional value, we can work on a different description.

Reply:
The idea behind using concrete yaml paths is that by browser searching it in a page it provides a high accuracy for relevant sentences especially in bigger files. Therefore I view it as providing high value for searching information of interest.

Reply:
Alright. Let us keep it, in that case.

Reply:
I agree